IGI.Lab3/CarPark.WebUI/Views/Home/Privacy.cshtml

```diff
@@ -0,0 +1,6 @@
+﻿@{
```

Comment:
Do you need this page?)

Reply:
This is not a mistake) For the future expansion of the project)

Reply:
Got it)